src/applications/caregivers/components/AdditionalInfo/index.jsx

```diff
@@ -29,45 +39,29 @@ const PrimaryCaregiverInfo = ({ additionalInfo, pageTitle }) => (
 
     {additionalInfo && (
       <section className="vads-u-margin-y--2p5">
-        <AdditionalInfo triggerText="What's a Primary Family Caregiver?">
+        <AdditionalInfo triggerText="What&apos;s a Primary Family Caregiver?">
```

Comment:
For what it's worth, I remember there was a consolidated effort once upon a time to change all straight apostrophe's to "fancy" ones: `’`

Line 53 has one of those "fancy" apostrophes.

Reply:
When I thought I found them all, went ahead and added the unicode for it, so it stays fancy!